src/libAtomVM/CMakeLists.txt

```diff
@@ -132,6 +132,11 @@ if (ADVANCED_TRACING)
     target_compile_definitions(libAtomVM PUBLIC ENABLE_ADVANCED_TRACE)
 endif()
 
+if(ENABLE_ASAN)
+    target_compile_options(libAtomVM PUBLIC -fsanitize=address -fno-omit-frame-pointer)
```

Comment:
Is `-fno-omit-frame-pointer` cross platform? I remember that was something quite common on i386 that has few available registers, but I think it is not the default on several platforms (and maybe it is not even available as an option on other archs).

Reply:
Good question. Logically, omitting frame pointer is an optimization so every platform should support that (making one less available register if disabled) but from reading GCC docs it's noop on archs that don't benefit from it.

https://gcc.gnu.org/onlinedocs/gcc-4.9.2/gcc/Optimize-Options.html 
look for `-fomit-frame-pointer`